unit/models/card.py

```diff
@@ -24,36 +24,60 @@ def from_json_api(_id, _type, attributes, relationships):
         )
 
 
-class BusinessDebitCardDTO(object):
-    def __init__(self, id: str, created_at: datetime, last_4_digits: str, expiration_date: str, ssn: str,
-                 full_name: FullName, date_of_birth: date, address: Address, phone: Phone, email: str,
-                 status: CardStatus, passport: Optional[str], nationality: Optional[str],
+class BusinessCardDTO(object):
+    def __init__(self, _id: str, _type: str, created_at: datetime, last_4_digits: str, expiration_date: str,
```

Comment:
added updated_at, the rest fields will be added soon.
let's discuss about enum

Reply:
OK so let's manage some TODO list..  I opened [this](https://docs.google.com/document/d/1mg71hM7j6K-PqwVsQSuZ6PMbr0lZzCMhTnS95ANhbQE/edit) let me know if you have another suggestion so it will be easy to track all these small debts:)